docs/resources/fgs_function.md

```diff
@@ -315,6 +315,17 @@ The following arguments are supported:
 
 * `log_stream_name` - (Optional, String) Specifies the name of the LTS stream.
 
+* `qualifier_name` - (Optional, String) Specifies the version name or alias name.
+
+-> If the parameter is alias name, The format is as follows **!xx**.
+
+* `reserved_instance_count` - (Optional, Int) Specifies the number of reserved instance.
+
+* `idle_mode` - (Optional, Bool) Specifies whether to enable the idle mode.
+
+* `tactics_config` - (Optional, List) Specifies the auto scaling policy for reserved instance.
+  The [object](#functiongraph_tactics_config) structure is documented below.
```

Comment:
```suggestion
  The [tactics_config](#functiongraph_tactics_config) structure is documented below.
```
Keep the name consistent, and the anchor points also need to be changed synchronously.